db/app/app.test.js

```diff
@@ -315,4 +355,29 @@ describe("PATCH", () => {
   });
 });
 
+describe("get /api/users", () => {
+  test("responds with status 200", () => {
+    return request(app).get("/api/users").expect(200);
+  });
+  test("rchecks whether each users has the correct keys", () => {
+    return request(app)
+      .get("/api/users")
+      .then((res) => {
```

Comment:
This `.forEach` needs to be guarded by a length condition. otherwise an unpopulated array will give a false positive.